src/process/WindowsLokinetProcessManager.cpp

```diff
@@ -14,7 +14,8 @@
 WindowsLokinetProcessManager::WindowsLokinetProcessManager()
 {
     ::CreateMutexA(nullptr, FALSE, "lokinet_qt5_ui");
-    path << ::getenv("SYSTEMROOT") << "\\System32\\net.exe";
+    path(::getenv("SYSTEMROOT"));
+    path.append("\\System32\\net.exe");

```

Comment:
`path /= fs::path{"System32"} / fs::path{"net.exe"}` i think would be better

Reply:
qt5 is c++11 and doesn't even know that `std::filesystem` exists at all